mobile/src/components/common/text/helpers.ts

```diff
@@ -0,0 +1,79 @@
+import { styles } from './style';
+
+export const selectFontColor = (color: string): object => {
+  let fontColor = {};
+
+  switch (color) {
+    case 'gray_400':
+      fontColor = styles.textGray_400;
+      break;
+    case 'gray_200':
+      fontColor = styles.textGray_200;
+      break;
+    case 'gray_100':
+      fontColor = styles.textGray_100;
+      break;
+    default:
+      fontColor = styles.textGray_400;
+      break;
+  }
+
+  return fontColor;
```

Comment:
Or you may add `fontColorToStyle` map and use it 🤔 
```suggestion
  switch (color) {
    case 'gray_400':
      return styles.textGray_400;
    case 'gray_200':
      return styles.textGray_200;
    case 'gray_100':
      return styles.textGray_100;
    default:
      return styles.textGray_400;
  }
```

Reply:
We don't need these helpers at all